### PR TITLE
Hackathon filter

### LINKED ISSUE #4117: Build - Filter Hackathon Prize
### User Story

As a hackathon participant I want to filter prize issues by sponsor/funder.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: Hackers want to see all issues posted by companies they are interested in. Sponsors want to be sure that their prizes can be found if people look for them.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
No way to filter issues by sponsor on the hackathon issue explorer.

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
Two ways to filter, which should have the same effect: a)Sidebar on hackathon issue explorer with names of all sponsors/funders b)Sponsor logos at the top of the page that can be clicked to filter.

Once filtered, only prizes posted by that sponsor should appear, and there should be some way of unfiltering that is easily accessible (so that hackers know there are other prizes available and can get back to the full list). This can involve unchecking a box on the sidebar, or clicking a link that says "See all prizes" somewhere.

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

- A way of clicking to filter by sponsor/funder
- Logos at the top of the hackathon page that can be clicked to filter
- A way of returning to an unfiltered view (see all prizes)

**Front End**
- Place logos on top
- Fetch Organization to build filter
- Send request when filter is clicked
- Logos will be manually added in the html page
- Check to hide top nav items: how it works, issue explorer, fund issue

**Estimate: 5**

hackathon

### PR DESCRIPTION
**Pull request by octavioamu**
##### Checklist

- [ ] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [ ] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

##### Refers/Fixes

Fixes #4117
###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: app/assets/v2/js/pages/dashboard.js [hunk 1]
```diff
@@ -374,13 +378,15 @@ var get_search_URI = function(offset, order) {
   if (!document.hackathon) {
     if (typeof order_by !== 'undefined') {
       uri += '&order_by=' + order_by;
+      uri += '&offset=' + offset;
+      uri += '&limit=' + results_limit;
     }
   } else {
-    uri += '&event_tag=' + document.hackathon;
+    uri += `&event_tag=${document.hackathon}`;
```
Comment (danlipert) [inline app/assets/v2/js/pages/dashboard.js]: why the backticks here instead of single quotes?

File: app/assets/v2/js/pages/dashboard.js [hunk 2]
```diff
@@ -477,9 +487,30 @@ var refreshBounties = function(event, offset, append, do_save_search) {
...
+        <input name="org" id="${org}" type="radio" value="${org}" val-ui="${org}" />
+        <label class="filter-label" for=${org}>
+          <img src="/dynamic/avatar/${org}" class="rounded-circle" width="24" height="24"> ${org}
+        </label>
+      </div>
+      `).join(' ')}
+      `);
+    }
+  };
+  // console.log(organ)
```
Comment (danlipert) [inline app/assets/v2/js/pages/dashboard.js]: remove comment here

File: app/assets/v2/js/pages/dashboard.js (hunk 1 above)
Comment (octavioamu) [reply app/assets/v2/js/pages/dashboard.js]: is a string literal

Comment (danlipert) [reply app/assets/v2/js/pages/dashboard.js]: ah my bad! thanks!

**Pull request merged**